build/install-build-tools.sh

```diff
@@ -46,12 +46,7 @@ echo "Dependency check: Java JDK 1.8.0_74"
 
 if [[ $($JAVA_HOME/bin/javac -version 2>&1) != "javac 1.8.0_74" ]]; then
     echo "WARN: Unable to find JDK 1.8.0_74, going to download it and set JAVA_HOME relative to ${PWD}"
-    if [[ -z ${JAVA_MIRROR} ]]; then
```

Comment:
Any special reason for removing JAVA_MIRROR option introduced in 075822949636e2458fba6b75ec40b66904f46f9b?